### DISCUSSION TITLE
neovoim extension with vscode

### DISCUSSION THREAD

**Discussion by Abdullah-Mahthel** [vscode-neovim/vscode-neovim#1312] [General]
![neovimxx](https://github.com/vscode-neovim/vscode-neovim/assets/130181169/3b2f8986-1a8a-47b2-90bd-832f89eada81)

can someone explain please :/

l am using Windows though

**Comment by RiteshChepuri** [comment]
you gotta use win32 for windows

**Comment by justinmk** [comment]
<img width="512" alt="image" src="https://github.com/user-attachments/assets/97922092-183c-422b-95a1-62065c4da6ef">


gets the following setting id:

    vscode-neovim.neovimExecutablePaths.win32
